src/components/views/UserHand.js

```diff
@@ -0,0 +1,139 @@
+
+    const WhiteCard = ({card}) => {
+        return(
+            <CardButton className={"card whiteCard simple"}
+                        children={card.text}
+                        key={card.id}
+            />
+        );
+    };
+    WhiteCard.propTypes = {
+        card: PropTypes.object
+    };
+
+    useEffect( () => {
+        async function getBlackCard() {
+            try {
+                const response = await api.get(`/users/${userId}/games/blackCards/current`,
+                    {
+                        headers: {
+                            "authorization": token
+                        }
+                    });
+                setBlackCard(response.data);
+            } catch (error) {
+                if (error.response.status === 401 || error.response.status === 404) {
+                    setBlackCard(null);
+                    console.error("Error ", error.response.status, ": ", error.response.message);
+                } else {
+                    console.error("Details: ", error);
+                    alert("Invalid Input:\n " + handleError(error));
+                }
+            }
+        }
+        getBlackCard();
+    }, []);
+
+    useEffect( () => {
+        async function getWhiteCards() {
+            try {
+                const response = await api.get(`/users/${userId}/games/whiteCards`,
+                    {
+                        headers: {
+                            "authorization": token
+                        }
+                    });
+                setWhiteCards(response.data);
+            } catch (error) {
+                if(error.response.status === 401 || error.response.status === 404) {
+                    setWhiteCards(null);
+                    console.error("Error " , error.response.status, ": ", error.response.message);
+                } else {
+                    console.error("Details: ", error);
+                    alert("Invalid Input:\n " + handleError(error));
+                }
+            }
+        }
+        getWhiteCards();
+    }, []);
+
+    let blackCardContent
+    let whiteCardsContent = <div className="hand">No more white Cards available</div>
+    let cardsOnPile;
+
+    if(blackCard) {
+        blackCardContent = <CardButton className="card blackCard" disabled={true}>
+            {blackCard.text}
+        </CardButton>
+    } else {
+        blackCardContent = <CardButton className="card blackCard"
+                                       onClick={() => {history.push('/game/select/blackCard')}}>
+            No black Card set, Choose black Card
+        </CardButton>
+    }
+
+    if (whiteCards){
+        if (whiteCards.length === 0 && !blackCard) {
+            whiteCardsContent = <CardButton className="card whiteCard simple"
+                                            onClick={() => {history.push('/game/select/blackCard')}}>
+                No white Card available, Did you choose a black Card?
+            </CardButton>
```

Comment:
It is a bit strange if the black card as on hover animation and the white card does not. can you add that for this white card?

Reply:
I wanted to do this this way because the white card should not apear to be clickable where as the black card should appear to be clickable.. yes I made the white card clickable as well but that is just an affordance but there does not need to be a signifier.. unless you want the white card to look disabled..